src/picom.c

```diff
@@ -136,38 +136,27 @@ static inline int64_t get_time_ms(void) {
 }
 
 // XXX Move to x.c
-void cxinerama_upd_scrs(session_t *ps) {
-	// XXX Consider deprecating Xinerama, switch to RandR when necessary
-	free_xinerama_info(ps);
-
-	if (!ps->o.xinerama_shadow_crop || !ps->xinerama_exists)
-		return;
-
-	xcb_xinerama_is_active_reply_t *active =
-	    xcb_xinerama_is_active_reply(ps->c, xcb_xinerama_is_active(ps->c), NULL);
-	if (!active || !active->state) {
-		free(active);
-		return;
-	}
-	free(active);
```

Comment:
quick research made me assume that if we have xrandr then it's "active" and there is no analog of this function in the xrandr extension

Reply:
agreed